Windows: tooling/perf-regression fails with CreateProcess error=193 (%1 not a valid Win32 application)

### Bug description

During the full build on Windows, the tooling/perf-regression module fails when PerfRegressionIT runs the perf-regression tool

```
Caused by: org.zeroturnaround.exec.InvalidExitValueException: Unexpected exit value: 1, allowed exit values: [0], executed command [java, -jar, target/quarkus-app/quarkus-run.jar, -d, 1s, 3.31.0-SNAPSHOT] with environment {LANG=en_US.UTF-8}, output was 4357 bytes:
__  ____  __  _____   ___  __ ____  ______ 
 --/ __ \/ / / / _ | / _ \/ //_/ / / / __/ 
 -/ /_/ / /_/ / __ |/ , _/ ,< / /_/ /\ \   
--\___\_\____/_/ |_/_/|_/_/|_|\____/___/   
2026-01-07 18:13:52,762 INFO  [io.quarkus] (main) perf-regression 3.31.0-SNAPSHOT on JVM (powered by Quarkus 3.30.1) started in 0.827s. 
2026-01-07 18:13:52,778 INFO  [io.quarkus] (main) Profile prod activated. 
2026-01-07 18:13:52,778 INFO  [io.quarkus] (main) Installed features: [cdi, picocli]
java.lang.RuntimeException: An issue occurred while attempting to execute 'mvnw help:evaluate -Dexpression='target-maven-version' -q -DforceStdout', more logs may be found in target\cq-versions-under-test\3.31.0-SNAPSHOT/logs.txt if exists                                                                     
        at org.apache.camel.quarkus.performance.regression.MvnwCmdHelper.execute(MvnwCmdHelper.java:88)
        at org.apache.camel.quarkus.performance.regression.PerfRegressionCommand.getTargetMavenVersion(PerfRegressionCommand.java:130)
        at org.apache.camel.quarkus.performance.regression.PerfRegressionCommand.runPerfRegressionForCqVersion(PerfRegressionCommand.java:110)
        at org.apache.camel.quarkus.performance.regression.PerfRegressionCommand.run(PerfRegressionCommand.java:78)
        at picocli.CommandLine.executeUserObject(CommandLine.java:2045)
        at picocli.CommandLine.access$1500(CommandLine.java:148)
        at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2469)
        at picocli.CommandLine$RunLast.handle(CommandLine.java:2461)
        at picocli.CommandLine$RunLast.handle(CommandLine.java:2423)
        at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2277)
        at picocli.CommandLine$RunLast.execute(CommandLine.java:2425)
        at io.quarkus.picocli.runtime.PicocliRunner$EventExecutionStrategy.execute(PicocliRunner.java:26)
        at picocli.CommandLine.execute(CommandLine.java:2174)
        at io.quarkus.picocli.runtime.PicocliRunner.run(PicocliRunner.java:40)
        at io.quarkus.runtime.ApplicationLifecycleManager.run(ApplicationLifecycleManager.java:141)
        at io.quarkus.runtime.Quarkus.run(Quarkus.java:80)
        at io.quarkus.runtime.Quarkus.run(Quarkus.java:51)
        at io.quarkus.runner.GeneratedMain.main(Unknown Source)
        at io.quarkus.bootstrap.runner.QuarkusEntryPoint.doRun(QuarkusEntryPoint.java:69)
        at io.quarkus.bootstrap.runner.QuarkusEntryPoint.main(QuarkusEntryPoint.java:37)
Caused by: java.io.IOException: Cannot run program "C:\Users\HOME\IdeaProjects\camel-quarkus\tooling\perf-regression\target\cq-versions-under-test\3.31.0-SNAPSHOT\mvnw" (in directory "target\cq-versions-under-test\3.31.0-SNAPSHOT"): CreateProcess error=193, %1 is not a valid Win32 application               
        at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1143)
        at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1073)
        at java.base/java.lang.Runtime.exec(Runtime.java:594)
        at org.apache.commons.exec.launcher.Java13CommandLauncher.exec(Java13CommandLauncher.java:48)
        at org.apache.commons.exec.launcher.CommandLauncher.exec(CommandLauncher.java:64)
        at org.apache.commons.exec.DefaultExecutor.launch(DefaultExecutor.java:513)
        at org.apache.commons.exec.DefaultExecutor.executeInternal(DefaultExecutor.java:334)
        at org.apache.commons.exec.DefaultExecutor.execute(DefaultExecutor.java:292)
        at org.apache.camel.quarkus.performance.regression.MvnwCmdHelper.execute(MvnwCmdHelper.java:79)
        ... 19 more
Caused by: java.io.IOException: CreateProcess error=193, %1 is not a valid Win32 application
        at java.base/java.lang.ProcessImpl.create(Native Method)
        at java.base/java.lang.ProcessImpl.<init>(ProcessImpl.java:499)
        at java.base/java.lang.ProcessImpl.start(ProcessImpl.java:158)
        at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1110)
        ... 27 more
2026-01-07 18:13:53,443 INFO  [io.quarkus] (main) perf-regression stopped in 0.055s
        at org.zeroturnaround.exec.InvalidExitUtil.checkExit(InvalidExitUtil.java:27)
        at org.zeroturnaround.exec.WaitForProcess.call(WaitForProcess.java:114)
        at org.zeroturnaround.exec.ProcessExecutor.waitFor(ProcessExecutor.java:1076)
        at org.zeroturnaround.exec.ProcessExecutor.execute(ProcessExecutor.java:906)
        at org.apache.camel.quarkus.performance.regression.it.PerfRegressionIT.nominalShouldPrintReport(PerfRegressionIT.java:96)
        ... 35 more

[ERROR] org.apache.camel.quarkus.performance.regression.it.PerfRegressionIT.noArgsShouldPrintHelp -- Time elapsed: 1.316 s <<< FAILURE!
java.lang.AssertionError: 

Expected: a string containing "-an, --also-run-native-mode"
     but: was "__  ____  __  _____   ___  __ ____  ______ 
 --/ __ \/ / / / _ | / _ \/ //_/ / / / __/ 
 -/ /_/ / /_/ / __ |/ , _/ ,< / /_/ /\ \   
--\___\_\____/_/ |_/_/|_/_/|_|\____/___/   
2026-01-07 18:13:54,582 INFO  [io.quarkus] (main) perf-regression 3.31.0-SNAPSHOT on JVM (powered by Quarkus 3.30.1) started in 0.828s. 
2026-01-07 18:13:54,591 INFO  [io.quarkus] (main) Profile prod activated. 
2026-01-07 18:13:54,592 INFO  [io.quarkus] (main) Installed features: [cdi, picocli]
Missing required parameter: '<versions>'
Usage: <main class> [-an] [-umnb] [-cqs=<cqStagingRepository>]
                    [-cs=<camelStagingRepository>]
                    [-d=<singleScenarioDuration>] <versions>...
Run a performance test against a list of Camel Quarkus versions and print a
report
      <versions>...   A list of versions, e.g: 2.7.0 2.8.0-SNAPSHOT
      -an, --also-run-native-mode
                      Tells whether the throughput test should also be run in
                        native mode. By default, run in JVM mode only.
      -cqs, --camel-quarkus-staging-repository=<cqStagingRepository>
                      Camel Quarkus staging repository, e.g: https://repository.
                        apache.org/content/repositories/orgapachecamel-1423
      -cs, --camel-staging-repository=<camelStagingRepository>
                      Camel staging repository, e.g: https://repository.apache.
                        org/content/repositories/orgapachecamel-1424
  -d, --duration=<singleScenarioDuration>
                      The duration of a single performance test scenario (e.g.
                        45s, 30m, 1h). Up to 2 scenarios per version could be
                        run.
      -umnb, --use-mandrel-native-builder
                      Tells whether mandrel should be used to build native
                        images. Can be used with camel-quarkus >= 2.8.0 only.
2026-01-07 18:13:54,786 INFO  [io.quarkus] (main) perf-regression stopped in 0.023s
"
        at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
        at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:6)
        at org.apache.camel.quarkus.performance.regression.it.PerfRegressionIT.noArgsShouldPrintHelp(PerfRegressionIT.java:54)
        at java.base/java.lang.reflect.Method.invoke(Method.java:568)
        at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
        at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)

[INFO] 
[INFO] Results:
[INFO] 
[ERROR] Failures: 
[ERROR]   PerfRegressionIT.noArgsShouldPrintHelp:54 
Expected: a string containing "-an, --also-run-native-mode"                                                                                               
     but: was "__  ____  __  _____   ___  __ ____  ______                                                                                                 
 --/ __ \/ / / / _ | / _ \/ //_/ / / / __/                                                                                                                
 -/ /_/ / /_/ / __ |/ , _/ ,< / /_/ /\ \                                                                                                                  
--\___\_\____/_/ |_/_/|_/_/|_|\____/___/                                                                                                                  
2026-01-07 18:13:54,582 INFO  [io.quarkus] (main) perf-regression 3.31.0-SNAPSHOT on JVM (powered by Quarkus 3.30.1) started in 0.828s.                   
2026-01-07 18:13:54,591 INFO  [io.quarkus] (main) Profile prod activated.                                                                                 
2026-01-07 18:13:54,592 INFO  [io.quarkus] (main) Installed features: [cdi, picocli]                                                                      
Missing required parameter: '<versions>'                                                                                                                  
Usage: <main class> [-an] [-umnb] [-cqs=<cqStagingRepository>]
                    [-cs=<camelStagingRepository>]
                    [-d=<singleScenarioDuration>] <versions>...
Run a performance test against a list of Camel Quarkus versions and print a
report
      <versions>...   A list of versions, e.g: 2.7.0 2.8.0-SNAPSHOT
      -an, --also-run-native-mode
                      Tells whether the throughput test should also be run in
                        native mode. By default, run in JVM mode only.
      -cqs, --camel-quarkus-staging-repository=<cqStagingRepository>
                      Camel Quarkus staging repository, e.g: https://repository.
                        apache.org/content/repositories/orgapachecamel-1423
      -cs, --camel-staging-repository=<camelStagingRepository>
                      Camel staging repository, e.g: https://repository.apache.
                        org/content/repositories/orgapachecamel-1424
  -d, --duration=<singleScenarioDuration>
                      The duration of a single performance test scenario (e.g.
                        45s, 30m, 1h). Up to 2 scenarios per version could be
                        run.
      -umnb, --use-mandrel-native-builder
                      Tells whether mandrel should be used to build native
                        images. Can be used with camel-quarkus >= 2.8.0 only.
2026-01-07 18:13:54,786 INFO  [io.quarkus] (main) perf-regression stopped in 0.023s
"
[ERROR]   PerfRegressionIT.nominalShouldPrintReport:116 The perf-regression process has finished with an unexpected exit value
[INFO] 
[ERROR] Tests run: 3, Failures: 2, Errors: 0, Skipped: 1
[INFO] 
```

--- a/tooling/perf-regression/src/main/java/org/apache/camel/quarkus/performance/regression/MvnwCmdHelper.java
+++ b/tooling/perf-regression/src/main/java/org/apache/camel/quarkus/performance/regression/MvnwCmdHelper.java
@@ -30,6 +30,7 @@ import org.apache.commons.exec.environment.EnvironmentUtils;
 import org.apache.commons.io.IOUtils;
 import org.apache.commons.io.output.ByteArrayOutputStream;
 import org.apache.commons.io.output.TeeOutputStream;
+import org.apache.commons.lang3.SystemUtils;
 import org.jboss.logging.Logger;
 
 public class MvnwCmdHelper {
@@ -43,7 +44,8 @@ public class MvnwCmdHelper {
         TeeOutputStream teeOutputStream = null;
 
         try {
-            File mvnwFile = cqVersionUnderTestFolder.resolve("mvnw").toFile();
+            String mvnWrapperExecutableName = SystemUtils.IS_OS_WINDOWS ? "mvnw.cmd" : "mvnw";
+            File mvnwFile = cqVersionUnderTestFolder.resolve(mvnWrapperExecutableName).toFile();
             CommandLine cmd = CommandLine.parse(mvnwFile.getAbsolutePath() + " " + args);
 
             stdoutAndStderrMemoryStream = new ByteArrayOutputStream();

